prow/jobs/generated/knative/func-main.gen.yaml

```diff
@@ -32,7 +32,7 @@ periodics:
         value: "true"
       - name: DOCKER_IN_DOCKER_ENABLED
         value: "true"
-      image: gcr.io/knative-tests/test-infra/prow-tests:v20221005-aef947d7
+      image: gcr.io/knative-tests/test-infra/prow-tests:v20220927-b6d8e28f
```

Comment:
This accidentally went backwards, but I expect automation will fix it forwards again.